js/app.js

```diff
@@ -24,11 +24,20 @@ Project.initializeProjects = function(){
   listProjects();
 }
 
+var navHandler = function(){
+  $('article').hide();
+  $('#homeBlock').show();
+  $('.tab').on('click', function(){
+    $('article').hide();
+    $('article[id="'+$(this).attr('data-content')+'"]').show();
```

Comment:
Remember that the selector for an id is `#the-id`, so you actually do not need such a complicated selector here.